xonsh/completers/base.py

```diff
@@ -11,6 +11,10 @@ def complete_base(prefix, line, start, end, ctx):
     and paths.  If we are completing the first argument, complete based on
     valid commands and python names.
     """
+    if line.strip() != "" and prefix != line:
```

Comment:
This would be better as `if line.strip() and prefix != line`

Reply:
done